packages/components/src/listbox/docs/Listbox.stories.mdx

```diff
@@ -135,18 +135,22 @@ However, think twice before adding *end* icons, *start* icons should be your go
             <Item key="earth">
                 <Text>Earth</Text>
                 <IconList slot="end-icon">
-                    <NotificationMajorIcon />
-                    <HelpMajorIcon />
-                    <LightbulbMajorIcon />
+                    <UfoMajorIcon />
+                    <TelescopeMajorIcon />
+                    <RocketMajorIcon />
                 </IconList>
             </Item>
             <Item key="jupiter">
                 <Text>Jupiter</Text>
-                <HelpMajorIcon slot="end-icon" />
+                <PlaceholderMajorIcon slot="end-icon" />
             </Item>
             <Item key="mars">
                 <Text>Mars</Text>
-                <LightbulbMajorIcon slot="end-icon" />
```

Comment:
What is the reason for the change here?

Reply:
Other list components docs example are all like this